.gitignore

```diff
@@ -133,4 +133,8 @@ dist
 .vscode
 
 # autosave draw.io
+.$models.drawio.bkp
+
+# misc
+.DS_Store
```

Comment:
Looks like it is something related to mac. Please add short comment to clarify this for other developers

Reply:
moved under #macOS section